### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Update steps for creating and deploying local catalogsource

Replace `Tekton-Pipelines` namespace with `openshift-pipelines-operator`

Update OLM install step

Signed-off-by: Nikhil Thomas <nikhilthomas1@gmail.com>

### CODE REVIEW

**Comment by nikhil-thomas** [comment]
@akihikokuroda, @vtereso 
could you guys review this PR

File: README.md [hunk 1]
```diff
@@ -69,22 +69,48 @@ docker push ${YOUR-REGISTORY}/openshift-pipelines-operator:${IMAGE-TAG}
...
+
+**Install OLM**
+
+ensure minikube is installed and docker env is set [see above](#install-minikube)
+
+```
+cd $GOPATH/src/github.com/operator-framework/operator-lifecycle-manager
+```
+```
+GO111MODULE=on NO_MINIKUBE=true make run-local
```
Comment (vtereso) [inline README.md]: I don't see a Makefile in this repo?

File: README.md [hunk 2]
```diff
@@ -69,22 +69,48 @@ docker push ${YOUR-REGISTORY}/openshift-pipelines-operator:${IMAGE-TAG}
...
 
 ```
 git clone git@github.com:operator-framework/operator-lifecycle-manager.git \
-          $GOPATH/github.com/operator-framework/
+          $GOPATH/src/github.com/operator-framework/
+```
+
+**Install OLM**
+
+ensure minikube is installed and docker env is set [see above](#install-minikube)
```
Comment (vtereso) [inline README.md]: Capitalize nit.

Comment (vtereso) [reply README.md]: There a few places this can be addressed, but it's not that important. I won't point them all out.

File: README.md (hunk 1 above)
Comment (nikhil-thomas) [reply README.md]: it is definitely needed. I shall make another PR.

Comment (sthaha) [reply README.md]: The `Makefile` refers to the one in `operator-lifecycle-manager`  see instruction above 
Makefile: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile

File: README.md (hunk 2 above)
Comment (nikhil-thomas) [reply README.md]: fixed

**Pull request merged**